packages/web-components/fast-components/src/data-grid/index.ts

```diff
@@ -11,29 +10,24 @@ import { DataGridStyles as gridStyles } from "./data-grid.styles";
 import { DataGridRowStyles as rowStyles } from "./data-grid-row.styles";
 import { DataGridCellStyles as cellStyles } from "./data-grid-cell.styles";
 
-const gridTemplate: ViewTemplate = createDataGridTemplate("fast");
-const rowTemplate: ViewTemplate = createDataGridRowTemplate("fast");
-const cellTemplate: ViewTemplate = createDataGridCellTemplate("fast");
-
 /**
- * The FAST Data Grid Element.
+ * The FAST Data Grid Cell Element.
  *
  * @public
```

Comment:
I recently added a "custom-elements.ts" file that exports all custom elements defined in the package. Can you add these components to that file? That will ensure the element is defined in storybook and for the fast-components.min.js files.

Reply:
done